src/collections/blog/2023/2023-05-31-interview-experience/index.mdx

```diff
@@ -0,0 +1,77 @@
+
+The tangible results of my contributions were evident. I successfully added the feature to deploy and undeploy Meshery adapters in Docker and Kubernetes is under progress. Moreover, I upgraded Meshsync CRD, making it configurable and enabling updates on specified resources with specified frequencies. The deprecation of Docker and compose CLIs in favor of Docker and compose Go SDKs streamlined internal command execution. Additionally, I resolved critical bugs, improving the stability and functionality of Meshery. Couldn't complete connection state machine but I hope that I'll deliver this too in the time I'm here. Finally, writing this blog served as another contribution to the project and the Meshery community.
+
+## V. Personal and Professional Growth:
+Professionally, this internship has been transformative. I have expanded my skill set significantly, gaining proficiency in Go programming and deep Kubernetes concepts. The experience has honed my problem-solving skills, adaptability, and resilience when working with unfamiliar technologies. Moreover, I have gained insights into project management, effective collaboration, and communication within a professional setting. Overall, this internship has had a profound impact on my career aspirations and has opened doors to future opportunities.
+
+## V. Conclusion
+
+   Overall it was a great experience. I recommend everyone to take part in similar programs. If you are a student or an early career professional go ahead and learn new languages, new technologies, meet new people, expand your horizons. If you're a veteran then help others out through your expertise, you can never know everything and it's never too late to learn.
+
+## VI. Connect
+
+   Follow me on <a href="https://twitter.com/AzanulZ"><img src="https://img.shields.io/badge/-Twitter-F3F7FA?logo=twitter&logoColor=1DA1F2&style=for-the-badge&logoWidth=30" alt="Twitter" /></a> & <a href="https://github.com/Azanul"><img src="https://img.shields.io/badge/-Github-F3F7FA?logo=github&logoColor=000000&style=for-the-badge&logoWidth=30" alt="Github" /></a>, lets connect on <a href="https://www.linkedin.com/in/azanul-haque"><img src="https://img.shields.io/badge/-Linkedin-F3F7FA?logo=linkedin&logoColor=0A66C2&style=for-the-badge&logoWidth=30" alt="Linkedin" /></a>.
+   Thanks for investing your time reading the post. I invite you to share your thoughts and experiences in the comments section.
+We are always looking for awesome people to join us, there is till a slot open for application [Meshery LFX mentorship](https://mentorship.lfx.linuxfoundation.org/project/0d6fd362-04a1-4086-a6e7-ec753ed4a60b). See you back on [Slack](http://slack.layer5.io/).
```

Comment:
Great blog @Azanul. Some cool words you have used. :cool: 
You can add a link to your member profile here instead which has all your social links.

Reply:
"Cool" must have been a word used in the chatgpt prompt.

Reply:
You guys seem to be referring to the only parts of the article that I have written 😜

Reply:
> Great blog @Azanul. Some cool words you have used. 🆒 You can add a link to your member profile here instead which has all your social links.

I think that'll be there regardless